packages/hunspell.rb

```diff
@@ -12,27 +13,24 @@ class Hunspell < Package
   is_fake
 
   if ARGV[0] == 'install'
-    puts
-    puts 'Enter your preferred language:'
-    puts '1 = American English'
-    puts '2 = Français'
-    puts '3 = Español'
-    puts '0 = Cancel'
-
-    while version = $stdin.gets.chomp
-      case version
-      when '1'
-        depends_on 'hunspell_en_us'
-        break
-      when '2'
-        depends_on 'hunspell_fr_fr'
-        break
-      when '3'
-        depends_on 'hunspell_es_any'
-        break
-      when '0'
-        abort
+    begin
+      @timeout = 10
+      @user_input = Timeout.timeout(@timeout) do
+        puts
+        puts 'Enter your preferred language:'
```

Comment:
Use heredoc?

Reply:
That makes it longer unnecessarily?

Reply:
It is worth to add two more line for a more clear look :)

Reply:
Please just suggest replacement code to do that. :)